core/src/main/java/tc/oc/pgm/spawns/states/Spawning.java

```diff
@@ -100,11 +100,13 @@ protected boolean trySpawn() {
   public void sendMessage() {}
 
   public void updateTitle() {
-    player.showTitle(
+    Title title =
         title(
             getTitle(),
             getSubtitle().color(NamedTextColor.GREEN),
-            Title.Times.of(Duration.ZERO, fromTicks(3), fromTicks(3))));
+            Title.Times.of(Duration.ZERO, fromTicks(3), fromTicks(3)));
+    player.showTitle(title);
+    player.getSpectators().forEach(p -> p.showTitle(title));
```

Comment:
This will display the same respawn title correct?
Maybe we could slightly alter the message based on the viewer, so instead of saying 
`Left click to respawn`

Either hide the subtitle for spectators or add a custom one to mention `{0} is respawning`

Although a majority of players will understand it's to mirror the target, someone may accidentally click or get confused 😆 

Just an idea to consider, well done @KingOfSquares 👍

Reply:
I thought about that as well, but could not think of something that would be as concise as the normal death message, which would also include info from the original death message (time until respawn, player can left click to spawn, can't respawn yet etc.)

Reply:
I like the " is respawning" one, will implement